File: front/eldo_app/lib/models/assistant.dart
class Assistant {
  late String id;
  late String shop;
  late String position;
  late String contacts;
  Assistant(
      {required this.id,
      required this.shop,
      required this.position,
      required this.contacts});

  Assistant.fromJson(Map<String, dynamic> json)
      : id = json['Id'],
        shop = json['Shop'],
        position = json['Position'],
        contacts = json['Contacts'];

  Map<String, dynamic> toJson() => {
        'id': id,
        'shop': shop,
        'position': position,
        'contacts': contacts,
      };
}

File: back/MobileApi/MobileApi/Models/Assistant.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace MobileApi.Models
{
    public partial class Assistant
    {
        public Assistant()
        {
            EvaluationSystems = new HashSet<EvaluationSystem>();
        }

        public Guid Id { get; set; }
        public string Shop { get; set; }
        public string Position { get; set; }
        public string Contacts { get; set; }

        public virtual User IdNavigation { get; set; }
        public virtual ICollection<EvaluationSystem> EvaluationSystems { get; set; }
    }
}
